website/templates/project/settings.mako

```diff
@@ -378,8 +378,31 @@
                     <div class="panel-heading clearfix">
                         <h3 class="panel-title">Email Notifications</h3>
                     </div>
+                    % if node['mailing_list_enabled'] or user['is_admin']:  ## Begin Project Mailing Lists Settings
```

Comment:
There are several display anomalies on the new project settings section:

1. Sorting icons are misaligned
2. Text overlaps with project title
3. The treebeard widget now has several extra columns (age actions etc); I'm not sure what the columns do, and they are not specified in the wireframes. They don't seem to update when I toggle mailing list settings either.
![screen shot 2016-05-20 at 1 40 35 pm](https://cloud.githubusercontent.com/assets/2957073/15436637/fa7b9a1c-1e90-11e6-8cdd-ea2030c53f14.png)